meta-mender-commercial/recipes-extended/images/mender-image-full-cmdline-rofs-commercial.bb

```diff
@@ -0,0 +1,3 @@
+require recipes-extended/images/mender-image-full-cmdline-rofs.bb
```

Comment:
@kacf This line introduces dependency between `meta-mender-commercial` layer and `meta-mender-demo` layer. Recipe `meta-mender-commercial/recipes-extended/images/mender-image-full-cmdline-rofs-commercial.bb` includes `meta-mender-demo/recipes-extended/images/mender-image-full-cmdline-rofs.bb`.
This dependency causes bitbake error for production build (without `meta-mender-demo` layer):
```
ERROR: ParseError at /my-project-dir/layers/meta-mender-commercial/recipes-extended/images/mender-image-full-cmdline-rofs-commercial.bb:1: Could not include required file recipes-extended/images/mender-image-full-cmdline-rofs.bb
```

Reply:
This is probably not as desired.

What do you think @kacf move the `rofs` recipe to `commercial` also?

Reply:
@oleorhagen @kacf Thanks! Now we can build with the latest meta-mender layer!
Issue is fixed in the following PRs:
- https://github.com/mendersoftware/meta-mender/pull/1891
- https://github.com/mendersoftware/meta-mender/pull/1892
- https://github.com/mendersoftware/meta-mender/pull/1893

Reply:
Great :smile: